qutebrowser/utils/utils.py

```diff
@@ -856,3 +856,47 @@ def expand_windows_drive(path):
         return path + "\\"
     else:
         return path
+
+
+def parse_number_sets(txtset, minnum, maxnum):
+    """Parse comma delimited numbers and ranges into a set.
+
+    Parse the given numbers set in text format, and return it as a list of
+    numbers, such that the numbers are never smaller than minnum, and never
+    greater than maxnum.
+
+    Args:
+        txtset: Number set specification. E.g. '1,5,first,8-last,2', where
+                'first' and 'last' are keywords denoting the minimum and
+                maximum allowable numbers respectively.
+        minnum: Minimum allowable number.
+        maxnum: Maximum allowable number.
+    """
+    txtset = txtset.lower() \
```

Comment:
Not sure what @The-Compiler's preference is here, but [PEP-8](https://www.python.org/dev/peps/pep-0008/#maximum-line-length) advises to do multi-line expressions via implicit continuations, i.e. wrapping the whole thing in parenthesis:

```python
txtset = (txtset.lower()
    .replace(...)
    .replace(...))
```

Reply:
I'd be happy to make this change. I've written very little Python, so I'm picking up the conventions as I go and have been relying on pylint to steer me the right direction. I imagine if the official style guide recommends parenthesis, then that's probably the way to go.

Reply:
I agree!